spec/lib/insert/insert_email_lists_spec.rb

```diff
@@ -33,7 +33,7 @@
                 deleted_result: "deleted correctly"
     }
     expect(result).to eq expected
-    expect(EmailList.find(email_lists[1])).to be_truthy
+    expect(EmailList.find(email_lists[1].id)).to be_truthy
```

Comment:
Previously we were calling `EmailList.find(some_EmailList_instance)`. That's deprecated in Rails 5

Reply:
Is this the same as `email_lists[1].reload` ?

Reply:
It is!

Reply:
I'll correct it.